code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -1,4 +1,4 @@
-#define MILK_TO_BUTTER_COEFF 15
+#define MILK_TO_BUTTER_COEFF 25
```

Comment:
we should make this a global define (place it in a file in the `_defines/` subdirectory) instead so you can tie this same define to the other line i commented on in this review.  this will prevent desynchronization in case someone decides to change the milk to butter coefficient to 20 or something in a weird balance PR if that matters in the future.

remember to remove the `#undef MILK_TO_BUTTER_COEFF` at the bottom of this file too

Reply:
This ALSO shows up as 25 in my machine, but i'm gonna have to do what you said anyways so